iina/Base.lproj/MiniPlayerWindowController.xib

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="14313.13.2" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES" customObjectInstantitationMethod="direct">
```

Comment:
Though I know that the xib file changes are not made by you intentionally, it should be generated by Xcode automatically, we should keep PR to be without these changes.